Tools/CMake/AMReXParallelBackends.cmake

```diff
@@ -222,29 +222,15 @@ if (AMReX_HIP)
          " Ensure that HIP is either installed in /opt/rocm/hip or the variable HIP_PATH is set to point to the right location.")
    endif()
 
-   if(${_this_comp} STREQUAL hipcc AND NOT AMReX_FORTRAN)
+   if(${_this_comp} STREQUAL hipcc)
        message(WARNING "You are using the legacy wrapper 'hipcc' as the HIP compiler.\n"
-           "This is only needed when building with Fortran support and with ROCm/HIP <=4.2.0. "
-           "Use AMD's 'clang++' compiler instead.")
+           "Use AMD's 'clang++'/'amdclang++' compiler, or on Cray "
+           "the CC compiler wrapper instead.")
    endif()
    # AMD's or mainline clang++ with support for "-x hip"
    # Cray's CC wrapper that points to AMD's clang++ underneath
    if(NOT ${_this_comp} STREQUAL hipcc)
        target_link_libraries(amrex PUBLIC hip::device)
-
-       # work-around for https://github.com/ROCm-Developer-Tools/HIP/issues/2278
-       # CXX_STANDARD always adds -std=c++XX, even if the compiler default fulfills it
-       #set_property(TARGET amrex PROPERTY CXX_STANDARD 17)
-       # note: already bumped to C++17 (cxx_std_17) or newer in AMReX_Config.cmake
-
-       # work-around for ROCm <=4.2
-       # https://github.com/ROCm-Developer-Tools/HIP/pull/2190
```

Comment:
Issue continues to exist and is now tracked here:
https://github.com/ROCm-Developer-Tools/hipamd/issues/12

Sheesh.